modules/k8s-demo-app/main.tf

```diff
@@ -20,7 +20,7 @@ resource "kubectl_manifest" "applications" {
   # For some reason using the above line returns a count not known until apply
   # error, even though the files are static. This needs to be kept in sync with
   # the YAML files defined in the services/ directory.
-  count     = 34
+  count     = 35
```

Comment:
the ingress GW artifact was not being installed unless this value was incremented. similar to https://github.com/hashicorp/terraform-aws-hcp-consul/pull/71